packages/vue/src/components/JsonForms.vue

```diff
@@ -182,6 +182,7 @@ export default defineComponent({
       return {
         data: this.jsonforms.core.data,
         errors: this.jsonforms.core.errors,
+        additionalErrors: this.jsonforms.core.additionalErrors,
```

Comment:
Why emit `additionalErrors`? We don't emit them in the other renderer sets. As the user already hands them in they should be aware of them already.